static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -922,18 +922,24 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
       value === AlertRuleComparisonType.CHANGE
         ? this.state.comparisonDelta ?? DEFAULT_CHANGE_COMP_DELTA
         : undefined;
-    const timeWindow = this.state.comparisonDelta
-      ? DEFAULT_COUNT_TIME_WINDOW
-      : DEFAULT_CHANGE_TIME_WINDOW;
+    const timeWindow =
+      value === AlertRuleComparisonType.COUNT
+        ? DEFAULT_COUNT_TIME_WINDOW
+        : DEFAULT_CHANGE_TIME_WINDOW;
     const sensitivity =
       value === AlertRuleComparisonType.DYNAMIC
         ? this.state.sensitivity || AlertRuleSensitivity.MEDIUM
         : undefined;
     const seasonality =
       value === AlertRuleComparisonType.DYNAMIC ? AlertRuleSeasonality.AUTO : undefined;
+    const thresholdType =
+      value === AlertRuleComparisonType.DYNAMIC
+        ? AlertRuleThresholdType.ABOVE_AND_BELOW
+        : AlertRuleThresholdType.ABOVE;
```

Comment:
let's combine these checks since the ? operator is adding complexity.

```
if (value === AlertRuleComparisonType.DYNAMIC) {
   ...
} else {
   ...
}
```

though, it seems like we're making a number of modifications based on the `value` here; it's best to use a `switch` for that kind of logic statement (it's both faster performance, and more readable)

```
switch(value) {
   case AlertRuleComparisonType.DYNAMIC:
       // handle case
       break;
   case AlertRuleComparisonType.CHANGE:
       // handle case
       break;
   default: 
      // if no case matches, falls to this
      break;
}
```